sway/commands/output/toggle.c

```diff
@@ -0,0 +1,37 @@
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/output.h"
+
+struct cmd_results *output_cmd_toggle(int argc, char **argv) {
+    if (!config->handler_context.output_config) {
+        return cmd_results_new(CMD_FAILURE, "Missing output config");
+    }
+
+    struct output_config *output = config->handler_context.output_config;
+
+    if (argc > 1 && strcmp(output->name, "*") == 0) {
+        return cmd_results_new(CMD_INVALID,
+                "Cannot apply toggle to all outputs.");
+    }
+
+    struct sway_output *s_output = all_output_by_name_or_id(output->name);
+
+    if(s_output == NULL){
+        return cmd_results_new(CMD_INVALID,
```

Comment:
Should be `CMD_FAILURE`

Reply:
I was looking at this as a reference https://github.com/swaywm/sway/blob/8c8e6a6c9089d70c3b970b03c3f783eafd16f825/sway/commands/output/transform.c#L48-L50

Reply:
There are some incorrect `CMD_INVALID` and `CMD_FAILURE` in the code base that need to be fixed. That is one of them. `CMD_INVALID` should be used for command syntax errors. `CMD_FAILURE` should be used for errors while trying to handle a syntactically valid command